### PR TITLE
add Software related descriptions

### PR DESCRIPTION
**Pull request by armintaenzertng**
This is an attempt to fill in the missing descriptions in the Software profile. Where possible, I tried to copy from the SPDX-2.3 specification.
Please note that this does not aim to be the final solution but will hopefully start discussions on possibly controversial points.

This misses descriptions for
- packageUrl
- contentIdentifier
- File

which I was not sure about their actual definition.

### CODE REVIEW

**Review by goneall — CHANGES_REQUESTED** [review]
A few suggested changes.

Thanks @armintaenzertng for kicking this off

File: model/Software/Classes/Sbom.md [hunk 1]
```diff
@@ -2,12 +2,16 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 # Sbom
 
+## Summary
+
+A container for a grouping of SPDX-3.0 content characterizing details about a software product.
```
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply model/Software/Classes/Sbom.md]: done

File: model/Software/Properties/contentType.md [hunk 2]
```diff
@@ -4,11 +4,12 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Provides information about the content type of an Element.
 
 ## Description
 
-A contentType is TODO
+This field is a reasonable estimation of the content type of the Element, from a developer perspective.
```
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply model/Software/Properties/contentType.md]: done

**Review by goneall — APPROVED** [review]
LGTM

**Comment by goneall** [comment]
Reviewed and approved on 28 Mar 2023 tech call

**Pull request merged**